revolv/templates/project/edit_project.html

```diff
@@ -132,4 +145,28 @@
         });
     });
 </script>
+<!-- Initializes the category selection tool -->
+<script src="{% static 'chosen/chosen.jquery.js' %}"></script>
+<script src="{% static 'chosen/docsupport/prism.js' %}" type="text/javascript" charset="utf-8"></script>
+<script>
+    $( document ).ready(function() {
+        // populates the currently selected categories if none of them have been selected
```

Comment:
Using the select multiple field instead of a hidden input, I can just set the initial value in the Class Based View, instead of doing hacky JS stuff!